subscribie/views.py

```diff
@@ -309,17 +297,30 @@ def on_complete_mandate():
 
         # Create subscription
         print("Creating subscription")
-        gocclient.subscriptions.create(
+        # Store Subscription against Person locally
+        person = database.session.query(Person).filter_by(email=session['email']).first()
+        subscription = Subscription(sku_uuid=session['package'], person=person)
+        database.session.add(subscription)
+        database.session.commit()
+
+        # Submit to GoCardless as subscription
+        gc_subscription = gocclient.subscriptions.create(
             params={
                 "amount": int(jamlaApp.sku_get_monthly_price(session["plan"])),
                 "currency": "GBP",
                 "name": jamlaApp.sku_get_title(session["plan"]),
                 "interval_unit": "monthly",
-                "metadata": {"sku": session["plan"]},
+                "metadata": {"subscribie_subscription_uuid": subscription.uuid},
```

Comment:
We now pass the uuid of the subscription stored on Subscribie's database to GoCardless as metadata. This makes it easier to track back the subscription to an sku. Note this is the uuid of the subscription record, not the uuid of the sku.  The database model maps subscriptions to sku's.